lib/Cleantalk/ApbctWP/Variables/Cookie.php

```diff
@@ -47,8 +47,19 @@ public function getVariable($name)
                             'apbct_site_landing_ts',
                             //'apbct_urls',
                             'apbct_timestamp',
-                            'apbct_site_referer')) ) {
+                            'apbct_site_referer'))
+                        ) {
                             $value = NoCookie::get($name);
+
+                            if ($name === 'apbct_site_referer' && !$value) {
+                                $value = AltSessions::get($name);
+                            }
+                            if ($name === 'apbct_prev_referer' && !$value) {
+                                $value = AltSessions::get($name);
+                            }
+                            if ($name === 'apbct_page_hits' && !$value) {
+                                $value = AltSessions::get($name);
+                            }
```

Comment:
Maybe shortener?
```
if (!$value && in_array($name, array(
'apbct_site_referer', 'apbct_prev_referer', 'apbct_page_hits'
)){
$value = AltSessions::get($name);
}
```

Reply:
Yeah, it's better https://github.com/CleanTalk/wordpress-antispam/pull/412/commits/e294bf736df4aade523a2cedfe5aa9b34a081fe4